.eslintrc.yml

```diff
@@ -10,4 +10,6 @@ globals:
 parserOptions:
   ecmaVersion: 2018
   sourceType: module
-rules: {}
+rules: {
+  linebreak-style: "off"
```

Comment:
There is no need to disable this line break.
you have to abide with the provided conventions from ESLINT.

Reply:
resolved